src/customTheme.ts

```diff
@@ -91,14 +122,17 @@ const customTheme = extendTheme(
           default: tertiaryTextLightMode,
           _dark: tertiaryTextDarkMode,
         },
-        primaryBg: {
-          default: 'gray.50',
-          _dark: 'gray.900',
-        },
-        secondaryBg: {
-          default: 'gray.100',
-          _dark: 'gray.800',
-        },
+        monochrome1,
+        monochrome2,
+        monochrome3,
+        monochrome4,
+        monochromeInverted1: invertSemanticToken(monochrome1),
+        monochromeInverted2: invertSemanticToken(monochrome2),
+        monochromeInverted3: invertSemanticToken(monochrome3),
+        monochromeInverted4: invertSemanticToken(monochrome4),
+        background1: invertSemanticToken(monochrome1),
+        background2: invertSemanticToken(monochrome2),
+        background3: invertSemanticToken(monochrome3),
```

Comment:
I added some handy monochrome semantic tokens with different levels and renamed the background tokens too. I think it makes more sense like this.

Reply:
This is really confusing me.

Can you explain what the big idea is here?

Reply:
The idea is to represent the monochrome color for light and dark themes on different levels.

Monochrome 1 is the lightest/darkest, while Monochrome 4 is the less light/dark. The idea is to be able to use these semantic tokens when we need to set something to black/white, not just the background (as we have been doing until now) but for other elements (like the GitHub link).

Reply:
In other words: in the same way, we don't directly use `gray.200` for the text color, but the corresponding `primaryText` token so it switches between light and dark variants automatically, these "monochrome" tokens are a shortcut following the same idea: set black/white variants for light and dark mode with different levels of tones.